src/uct/ib/mlx5/ib_mlx5.h

```diff
@@ -36,9 +36,9 @@
 #  include <infiniband/mlx5dv.h>
 #else
 #  include <infiniband/mlx5_hw.h>
-#  include "ib_mlx5_hw.h"
+#  include "exp/ib_mlx5_hw.h"
 #endif
-#include "ib_mlx5_dv.h"
+#include "dv/ib_mlx5_dv.h"
```

Comment:
don't we need some condition to include this? what happens on mofed?

Reply:
No, uct_ib_mlx5dv_init_obj() API we provide unconditionally.
If MOFED doesn't provide underlying  mlx5dv_init_obj() we emulate it using mlx5_hw.h